packages/beasties/src/index.ts

```
@@ -139,13 +56,14 @@
     const fs = this.fs
-    return new Promise((resolve, reject) => {
-      const callback = (err, data) => {
+    return new Promise<string>((resolve, reject) => {
+      // eslint-disable-next-line node/prefer-global/buffer
+      const callback = (err: NodeJS.ErrnoException | null, data: string | Buffer) => {
         if (err)
           reject(err)
-        else resolve(data)
+        else resolve(data.toString())
       }
       if (fs && fs.readFile) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)